apps/gpc-verifier/README.md

```diff
@@ -6,11 +6,11 @@ First, run the development server:
 yarn dev
 ```
 
-Open [http://localhost:3001](http://localhost:3001) with your browser to see the result.
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
```

Comment:
Is this file useful?  Should it be deleted, or made into a simple reference to the top-level README?

Reply:
not useful, will delete